### DISCUSSION TITLE
factur'x xml extraction in command line

### DISCUSSION THREAD

**Discussion by lubinfo** [sumatrapdfreader/sumatrapdf#3574] [Ideas]
Bonjour,

Les fichiers Factur'x contiennent un fichier .xml dans le .pdf.
Est-il possible de rajouter la fonctionnalité d'extraction de se fichier en ligne de commande ?

Factur'x files contain an .xml file in the .pdf.
Is it possible to add the functionality of extracting files from the command line?

**Comment by GitHubRulesOK** [comment]
Hmm thats unlikely
there are many good existing command line tools with extractors such as mutool/poppler and others without the needs of a viewer.

If you need to view and extract then SumatraPDF could run those external extractors , but why?.
